internal/logger/logger.go

```diff
@@ -34,6 +34,11 @@ func Red() PrintFunc {
 }
 
 func envColor(env string, defaultColor color.Attribute) color.Attribute {
+	_, err := strconv.Atoi(os.Getenv("FORCE_COLOR"))
+	if err == nil {
+		color.NoColor = false
+	}
```

Comment:
Hi @automation-stack,

I believe that we can just check for the existence of the key:

```suggestion
	if os.Getenv("FORCE_COLOR") != "" {
		color.NoColor = false
	}
```

Also, can you add an entry to the CHANGELOG and the API page on the documentation?

- https://github.com/go-task/task/blob/master/CHANGELOG.md#unreleased
- https://github.com/go-task/task/blob/master/docs/docs/api_reference.md#env
- https://taskfile.dev/api/#env